fw-child/template/glossary.php

```diff
@@ -14,13 +15,13 @@
 $glossary_query_args = array(
 	'post_type'      => 'definition',
 	'posts_per_page' => -1,
-	'order'          => 'asc'
+	'order'          => 'asc',
+	'orderby'		 => 'title'
```

Comment:
Tabs are used for spacing before the `=` instead of spaces (contrary to the other lines)

![image](https://github.com/CanadianClimateDataPortal/climatedata-wp-theme/assets/159493515/9c1baf9e-1ca5-45f0-8820-6b036bc8db67)